app/views/layouts/_analytics.html.erb

```diff
@@ -0,0 +1,17 @@
+<!-- Google Analytics -->
+<script>
+
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+ga('create', <%= ENV['UA_TRACKING_ID'] %>, 'auto');
```

Comment:
load this from Rails config rather than directly from `ENV`

Reply:
Ahhhh yes. My bad